web/styles/settings.css

```diff
@@ -226,6 +226,7 @@ h3,
     /* Limit the actions column to not using excessive width */
     .admin-table-wrapper table.admin_profile_fields_table tr .actions {
         width: 11%;
+        min-width: 70px;
```

Comment:
This is needed to avoid the actions column taking extra space and make the width (almost) same as it was before by overriding the `min-width: 100px` set in general for all heading using `thead th` in [this commit](https://github.com/zulip/zulip/commit/7ccfed65e8fb614275c4ba16c109d3352845ba76).

I think we should actually remove the extra space on `Actions` column from other tables as well. The `min-width` property was added to not change width of columns when sorting the tables by clicking on heading cells, but we anyway not allow sorting from `Actions` column.

Reply:
Probably reasonable to propose that in a separate PR and have @karlstolley review.